stencil.config.ts

```diff
@@ -12,51 +12,51 @@ export const create: () => Config = () => ({
   namespace: "calcite",
   bundles: [
     { components: ["calcite-accordion", "calcite-accordion-item"] },
-    { components: ["calcite-action"] },
     { components: ["calcite-action-bar"] },
-    { components: ["calcite-action-menu"] },
```

Comment:
Is the menu still here?

Reply:
Might have been a merge issue as this PR shouldn’t have touched this component.

Reply:
menu will be added in a separate PR.

Reply:
@anveshmekala calcite-action-menu already exists.

Reply:
my bad, thought it was `calcite-menu`. added it back.